_sass/_objects--typography.scss

```diff
@@ -0,0 +1,3 @@
+body {
```

Comment:
@TheMightyPenguin you haven't renamed this file please make sure to satisfy all requested changes before asking another review. ;)